torch/ao/quantization/backend_config/onednn.py

```diff
@@ -119,47 +119,58 @@ def _fuse_linear_bn_leaky_relu(is_qat, linear, bn, leaky_relu):
 observation_type = ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT
 linear_configs = _get_linear_configs(linear_dtype_configs)
 
-# (1) Linear + leaky_relu
-# -------------------
-# 1.1 linear module + leaky_relu fusion config
-# linear leaky_relu, linear module + leaky_relu module
-linear_configs.append(
-    BackendPatternConfig((nn.LeakyReLU, nn.Linear))
-        .set_dtype_configs(linear_dtype_configs)  # noqa: E131
-        .set_fuser_method(_sequential_wrapper2(nni.LinearLeakyReLU))
-        .set_fused_module(nni.LinearLeakyReLU))
-# linear leaky_relu, linear module + functional leaky_relu
-linear_configs.append(
-    BackendPatternConfig((F.leaky_relu, nn.Linear))
-        .set_dtype_configs(linear_dtype_configs)  # noqa: E131
-        .set_fuser_method(_sequential_wrapper2(nni.LinearLeakyReLU))
-        .set_fused_module(nni.LinearLeakyReLU))
-# linear leaky_relu, linear module + BN + leaky_relu
+def _add_eltwise_fusion_configs(configs, root_module, root_op, post_module, post_op,
+                                dtype_configs, fuser_method, fused_module, observation_type,
+                                ref_quant_module):
+    # 1 base module + op module fusion config
+    configs.append(
+        BackendPatternConfig((post_module, root_module))
```

Comment:
nit: the pattern need to be updated since we recently landed: https://github.com/pytorch/pytorch/pull/90698

Reply:
Ok. It's fixed.